posthog/api/test/test_person.py

```diff
@@ -88,44 +97,89 @@ def test_filter_by_cohort(self):
 
         cohort = Cohort.objects.create(team=self.team, groups=[{"properties": {"$os": "Chrome"}}])
         cohort.calculate_people()
-        response = self.client.get("/api/person/?cohort=%s" % cohort.pk).json()
-        self.assertEqual(len(response["results"]), 1, response)
+        response = self.client.get(f"/api/person/?cohort={cohort.pk}")
+        self.assertEqual(len(response.json()["results"]), 1, response)
+
+    def test_filter_person_list(self):
+
+        person1: Person = Person.objects.create(
+            team=self.team, distinct_ids=["distinct_id", "another_one"], properties={"email": "someone@gmail.com"},
+        )
+        person2: Person = Person.objects.create(
+            team=self.team, distinct_ids=["distinct_id_2"], properties={"email": "another@gmail.com"},
+        )
+
```

Comment:
would be great to add a test to explicitly make sure we don't see PII from other teams

Reply:
Definitely! I'll add this before the next review

Reply:
Ready for review